modules/tinymce/src/plugins/help/main/ts/data/PluginUrls.ts

```diff
@@ -6,65 +6,66 @@
  */
 
 const urls = [
-  { key: 'advlist', name: 'Advanced List' },
-  { key: 'anchor', name: 'Anchor' },
-  { key: 'autolink', name: 'Autolink' },
-  { key: 'autoresize', name: 'Autoresize' },
-  { key: 'autosave', name: 'Autosave' },
-  { key: 'bbcode', name: 'BBCode' },
-  { key: 'charmap', name: 'Character Map' },
-  { key: 'code', name: 'Code' },
-  { key: 'codesample', name: 'Code Sample' },
-  { key: 'colorpicker', name: 'Color Picker' },
-  { key: 'directionality', name: 'Directionality' },
-  { key: 'emoticons', name: 'Emoticons' },
-  { key: 'fullpage', name: 'Full Page' },
-  { key: 'fullscreen', name: 'Full Screen' },
-  { key: 'help', name: 'Help' },
-  { key: 'hr', name: 'Horizontal Rule' },
-  { key: 'image', name: 'Image' },
-  { key: 'imagetools', name: 'Image Tools' },
-  { key: 'importcss', name: 'Import CSS' },
-  { key: 'insertdatetime', name: 'Insert Date/Time' },
-  { key: 'legacyoutput', name: 'Legacy Output' },
-  { key: 'link', name: 'Link' },
-  { key: 'lists', name: 'Lists' },
-  { key: 'media', name: 'Media' },
-  { key: 'nonbreaking', name: 'Nonbreaking' },
-  { key: 'noneditable', name: 'Noneditable' },
-  { key: 'pagebreak', name: 'Page Break' },
-  { key: 'paste', name: 'Paste' },
-  { key: 'preview', name: 'Preview' },
-  { key: 'print', name: 'Print' },
-  { key: 'save', name: 'Save' },
-  { key: 'searchreplace', name: 'Search and Replace' },
-  { key: 'spellchecker', name: 'Spell Checker' },
-  { key: 'tabfocus', name: 'Tab Focus' },
-  { key: 'table', name: 'Table' },
-  { key: 'template', name: 'Template' },
-  { key: 'textcolor', name: 'Text Color' },
-  { key: 'textpattern', name: 'Text Pattern' },
-  { key: 'toc', name: 'Table of Contents' },
-  { key: 'visualblocks', name: 'Visual Blocks' },
-  { key: 'visualchars', name: 'Visual Characters' },
-  { key: 'wordcount', name: 'Word Count' },
+  { key: 'advlist', name: 'Advanced List', slug: 'opensource/advlist' },
```

Comment:
I'm a little concerned about the code bloat of this change. I'll take a look later to see if there's something better we can do.

Reply:
ok, feel free to close this PR and create a TINY ticket to resolve it properly if there is a better solution.